1_send_data/arduino_secrets.h

```diff
@@ -9,4 +9,4 @@ const char SECRET_APN_PASS[] = "";		// specify the password for your APN (if nee
 	// You API key must have at least the predefined 'MQTT Device' rights profile
 	// (alternatively: 'Device Access' read + write rights if need to customise the rights).
 	// Please note that you *must* use a TLS connection (MQTTS) if you grant more rights to the API key.
-const char SECRET_LIVEOBJECTS_API_KEY[] = "...";
+const char SECRET_LIVEOBJECTS_API_KEY[] = "bf4ef02c497345128ecda296f15e9f02";

```

Comment:
Tell me it is not a mistake and this is not your actual API key ;)
We should be consistent on credential example, either we put a sample value or we just leave them empty - but I couldn't decide for the API key and left "...". Moreover, I put a [verification routine in the .h file](https://github.com/mdelain/Arduino_MKR_NB_1500/blob/master/Simple_parameter/LiveObjects.h#L487) (code stops if credential != 32).
Arduino examples usually leave them empty.

Reply:
Of course, it was just "example" key ;)